### PR TITLE
Added wider temperature and power range, added further checks

### PR DESCRIPTION
**Pull request by MKesenheimer**
Since my power production was higher than 600W, I increased the max_value for power. Furthermore, I added more checks when adding the energy value to the daily production.

### CODE REVIEW

File: inverter/definitions/deye_2mppt_validations.yaml
Comment (jedie) [inline inverter/definitions/deye_2mppt_validations.yaml:1]: I add these changes here: https://github.com/jedie/inverter-connect/pull/118

File: inverter/publish_loop.py [hunk 1]
```diff
@@ -49,7 +49,9 @@ def publish_forever(*, config: Config, verbosity):
                             elif isinstance(value.value, Version):
                                 ha_value = str(value.value)
 
-                            daily_production_reset(value)
+                            if value.name == 'Total Power' and value.value != 0:
```
Comment (jedie) [inline inverter/publish_loop.py:52]: These are made in DailyProductionReset.__call__

File: Dockerfile
Comment (jedie) [inline Dockerfile:1]: Nice... But i don't know why a Docker setup is needed here... We use already a own virtualenv and it should work for a wide range of Python versions...

File: cfg/inverter-connect-template.toml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+exec_start = "/home/user/inverter-connect/.venv-app/bin/python -m inverter publish-loop"
+syslog_identifier = "inverter_connect"
+config_path = "/home/user/.config/inverter-connect"
+
+[mqtt]
+# MQTT server settings.
+host = "192.168.2.28"
+port = 1883
+user_name = "mqtt"
+password = "<i-am-not-telling>"
```
Comment (jedie) [inline cfg/inverter-connect-template.toml:25]: That's the problem to add a hardcoded settings file ;)

File: config.yaml
Comment (jedie) [inline config.yaml:1]: what is the purpose of this file?

**Comment by MKesenheimer** [comment]
Sorry, the other commits were not intended to be in this pull request. Please ignore them.

The `Dockerfile` and the `config.yaml` are for a addon that I built for Home Assistant. With these changes, your tool can be added directly as a plugin to Home Assistant which pulls the values and publishes them to a MQTT server running in Home Assistant.

**Comment by MKesenheimer** [comment]
I am closing this PR since you already implemented the relevant changes. Thank you.

**Comment by jedie** [comment]
> The Dockerfile and the config.yaml are for a addon that I built for Home Assistant. With these changes, your tool can be added directly as a plugin to Home Assistant which pulls the values and publishes them to a MQTT server running in Home Assistant.

That's interesting. Can you provide more information about this?
I use Home Assistant, too. But run inverter-connect on a different machine and use Mosquitto for as MQTT broker...

**Comment by MKesenheimer** [comment]
There is the possibility in Home Assistant to import custom addons. If I remember correctly, the addons are stored in the directory ´/root/addons´. 
You can simply copy your project folder via ssh or a samba share into this directory. After a restart of HA, your addon should be visible under ´Settings > Add-ons > Add-on Store´. The custom addon can be installed like every other addon by clicking "install". If everything went correctly, HA should now run an instance of "inverter-connect". Keep in mind, that the mqtt broker must be installed separately.

**Comment by MKesenheimer** [comment]
Oh, one more thing: I don't know yet, how to modify configuration files via the HA interface. Therefore, if you want to try out my changes, you have to modify the file `cfg/inverter-connect-template.toml` by hand and rename it to `cfg/inverter-connect.toml`. This file is copied automatically during the Docker build.

**Comment by jedie** [comment]
Thank for the information... Hm. Manually change the settings file is bad :(
I also don't know if this is in general a good idea here, because i use HomeAssistant via YunoHost...

We should move this into a discussion ;)

**Pull request closed without merging**